osu.Game/Screens/Edit/Verify/VisibilitySection.cs

```diff
@@ -26,6 +26,8 @@ internal class VisibilitySection : EditorRoundedScreenSettingsSection
         [BackgroundDependencyLoader]
         private void load(OverlayColourProvider colours)
         {
+            var hiddenIssueTypes = verify.HiddenIssueTypes.GetBoundCopy();
```

Comment:
you'll have to store this to a field. bindables which don't have a reference are automatically cleaned up (all bindings are tracked using weak references internally).